run.sh

```diff
@@ -1,30 +1,45 @@
 #!/bin/bash
-set -euxo pipefail
+set -euo pipefail
+
+function outputResult() {
+  local result=$1
+  local exitCode=$2
+
+  echo $result
+
+  if [ $OUTPUT_RESULT = true ]; then
+    echo "::set-output name=result::$result"
+  else
+    exit $exitCode
```

Comment:
…this should be
```suggestion
    [ $result = passed ]
    exit $?
```
or something like that.

Reply:
i think how i've changed it should be fine?

Reply:
Works. I think it could be clearer.